demo

```diff
@@ -67,4 +75,56 @@ exec docker-compose \
      -f docker-compose.storage.minio.yml \
      -f docker-compose.demo.yml \
      $CLIENT \
-     "$@"
+     "$@" &
```

Comment:
You will need to check whether `up` is one of the arguments passed, because `demo` script can be used for any docker-compose command, and especially `down` is frequently used. Something aka `echo "$@" | grep '\bup\b'` maybe?